tests/test_sms.py

```diff
@@ -0,0 +1,31 @@
+"""Test Twilio SMS service"""
+
+import time
+
+from twilio.rest.api.v2010.account.message import MessageInstance
+
+from how2meet.services.sms import send_sms
+
+
+def test_send_sms():
```

Comment:
Good on you for writing test, I often don't even make enough time for that in my real job 😭

Reply:
Hahah I'm trying to treat this as a "portfolio project" to showcase understanding and capability of a lot of skills:
- CI/CD and IaC
- APIs
- UI/UX
- Planning and documentation
- Linting, formatting, typechecking (best practices)
- And testing 

So even if the tests are fairly straightforward, I would really like to try and keep up with them. Easier to do it as you go than add them later. Plus, we're not really under time pressure so why not? lol

Reply:
Also also, ChatGPT makes writing tests take like 2 seconds so the overhead is pretty minimal!